application/helpers/lib_helper.php

```diff
@@ -107,15 +84,11 @@ function load_db_team() {
 }
 
 /*
- * This function gets the cached team data.
- * If cookie is deleted, get from the database.
+ * This function gets the cached team data from database.
  */
 if (! function_exists('load_cached_team')) {
     function load_cached_team() {
-        $data = load_cookie_team();
```

Comment:
OK, we can skip the php implementation if it cost too much. Actually, I have no experience on using localStorage personally. Maybe @Luolc can add more comments?